flask/cluster.py

```diff
@@ -2,11 +2,15 @@
 import subprocess
 import utils
 import query
+from sys import platform
 
 
 uclust_identity = utils.get_config()['uclust_identity'] # how similar sequences in the same cluster must be
 sequences_filename = 'usearch/sequences.fsa'
-usearch_binary_filename = 'usearch/usearch10.0.240_i86linux32'
+if platform == "linux" or platform == "linux2":
+    usearch_binary_filename = 'usearch/usearch10.0.240_i86linux32'
+elif platform == "darwin":
+    usearch_binary_filename = 'usearch/usearch11.0.667_i86osx32'
```

Comment:
How should we handle when platform doesn't equal any of these strings?  Maybe add an else that reports the error?

Reply:
See ce785166be8d97a4894ab25ed46bec244b77d12d